app/Libraries/MultiDB.php

```diff
@@ -107,9 +109,35 @@ public static function checkDomainAvailable($subdomain): bool
         return true;
     }
 
+    public static function checkExpenseMailboxAvailable($expense_mailbox): bool
+    {
+
+        if (!config('ninja.db.multi_db_enabled')) {
+            return Company::where("expense_mailbox", $expense_mailbox)->withTrashed()->exists();
```

Comment:
I don't think withTrashed() is valid on the company model.

Reply:
i initialy have used this, to ensure each expense_mailbox is unique. but I think you right, its not required